README.md

```diff
@@ -93,10 +83,69 @@ $ rake ids:update
 This rake task queries the NRDB API and creates/updates identities as appropriate.
 Identities not in the database are stripped out of ABR uploads to avoid errors.
 
+## Deploy as a web server
+- Get the project
+```shell
+git clone https://github.com/muyjohno/cobra.git
+cd cobra
+```
+- Set up config files
+```shell
+cp config/database.example.yml config/database.yml
+cp config/secrets.example.yml config/secrets.yml
+echo "RAILS_ENV=production" > .env
+echo "COMPOSE_FILE=prod" >> .env
+echo "POSTGRES_PASSWORD=some-good-password" >> .env
+echo "SECRET_KEY_BASE=random-64-bit-hex-key" >> .env
+echo "COBRA_DOMAIN=cobr.ai" >> .env
+echo "NISEI_DOMAIN=tournaments.nisei.net" >> .env
+```
+- Deploy
+```shell
+bin/deploy
+```
+
+## Deploy with Pulumi
+
+The deploy directory contains scripts for deploying to DigitalOcean using Pulumi for infrastructure as code. Here are
+some steps for setting that up.
+
+1. Fork the GitHub repository.
+2. Set a Pulumi access token and a DigitalOcean token in GitHub secrets, PULUMI_ACCESS_TOKEN and DIGITALOCEAN_TOKEN.
+   You can get these from the Pulumi and DigitalOcean websites, see their documentation.
+3. In the deploy directory, log into Pulumi CLI and create a new stack.
+4. Run this in the deploy directory: `pulumi config set cobra:cobra_domain your_domain.com`. 
+   Ensure you own the domain you want to use.
+5. Do the same for cobra:nisei_domain. If you don't like the defaults for cobra:region and cobra:size, you can set them
+   to slug values shown here: https://slugs.do-api.dev/.
+6. Check in the resulting Pulumi.stackname.yaml file to Git. If you prefer not to put the domain names in that file,
```

Comment:
what's the problem here?  Are there any actual secrets logged there or just the names?

Reply:
No, just the domain names. I've just removed this stuff, it seems pretty irrelevant.